src/main/java/com/example/sample/application/JakartaConcurrencyService.java

```diff
@@ -8,7 +8,7 @@
 * Contributors:
 *     IBM Corporation - initial API and implementation
 *******************************************************************************/
-package io.openliberty.sample.application;
+package com.example.sample.application;
 
```

Comment:
Refactoring from `io.openliberty` to `com.example` since this isn't hosted in the Open Liberty repository.